recipe/conda_build_config.yaml

```diff
@@ -0,0 +1,5 @@
+# https://conda-forge.org/docs/maintainer/knowledge_base.html#requiring-newer-macos-sdks
+
+# Ice 3.6.5 includes TLS 1.3 support which requires a more recent SecureTransport than the default 10.9 SDK
+MACOSX_DEPLOYMENT_TARGET:  # [osx]
+  - 10.13                  # [osx]
```

Comment:
As far as I can tell it's a hard requirement for Ice 3.6.5, I couldn't find a way to get it to compile without TLS 1.3. Older versions of Ice (e.g. 3.6.4) which don't support TLS 1.3 should compile, but it'd be preferable to keep TLS 1.3 support since in general operating systems are getting stricter about support for older SSL/TLS and any ciphers.

Reply:
We do package `openssl` on macOS. Unsure if there is a way to configure it to use `openssl` or not.